src/LanguageServer/Impl/Implementation/Server.cs

```diff
@@ -154,17 +154,20 @@ public sealed partial class Server : IDisposable {
             return Task.CompletedTask;
         }
 
-        public async Task DidChangeConfiguration(DidChangeConfigurationParams @params, CancellationToken cancellationToken) {
+        public void DidChangeConfiguration(DidChangeConfigurationParams @params, CancellationToken cancellationToken) {
             _disposableBag.ThrowIfDisposed();
-
-            var reanalyze = true;
-            if (@params.settings != null) {
-                if (@params.settings is ServerSettings settings) {
-                    reanalyze = HandleConfigurationChanges(settings);
-                } else {
-                    _log?.Log(TraceEventType.Error, "change configuration notification sent unsupported settings");
+            switch (@params.settings) {
+                case null:
```

Comment:
No logging for `null`?

Reply:
Null means it is missing so it is not `unsupported`